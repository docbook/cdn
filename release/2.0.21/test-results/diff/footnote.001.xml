<html xmlns:deltaxml="http://www.deltaxml.com/ns/well-formed-delta-v1" xmlns="http://www.w3.org/1999/xhtml" xmlns:dxx="http://www.deltaxml.com/ns/xml-namespaced-attribute" xmlns:dxa="http://www.deltaxml.com/ns/non-namespaced-attribute" deltaxml:deltaV2="A!=B" deltaxml:version="2.0" deltaxml:content-type="full-context">
   <head deltaxml:deltaV2="A=B">
      <link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css"/>
      <link rel="stylesheet" type="text/css" href="../../resources/css/default.css"/>
      <link href="../../resources/css/prism.css" rel="stylesheet" type="text/css"/>
      <meta charset="utf-8"/>
      <meta name="generator" content="DocBook XSL 2.0 Stylesheets Vxxx"/>
      <script type="text/javascript" src="../../resources/js/dbmodnizr.js"/>
      <title>Article wrapper</title>
   </head>
   <body deltaxml:deltaV2="A!=B">
      <article deltaxml:deltaV2="A!=B" id="R.1" class="article">
         <header deltaxml:deltaV2="A=B" class="article-titlepage">
            <h2>Article wrapper</h2>
         </header>
         <div deltaxml:deltaV2="A=B" class="lists-of-titles">
            <div class="toc">
               <p>
                  <b>Table of Contents</b>
               </p>
               <ul class="toc">
                  <li>
                     <span>
                        <a href="#R.1.3">First Section</a>
                     </span>
                  </li>
                  <li>
                     <span>
                        <a href="#R.1.4">Section Title</a>
                     </span>
                     <ul class="toc">
                        <li>
                           <span>
                              <a href="#R.1.4.3">Subsection Title</a>
                           </span>
                        </li>
                     </ul>
                  </li>
               </ul>
            </div>
         </div>
         <p deltaxml:deltaV2="A">This paragraph contains a footnote<sup class="footnote">
               <span class="osq">[</span>
               <a id="R.1.2.1" href="#ftn.R.1.2.1">1</a>
               <span class="csq">]</span>
            </sup>.</p><div deltaxml:deltaV2="B" class="content">
            <p>This paragraph contains a footnote<sup class="footnote">
                  <span class="osq">[</span>
                  <a id="R.1.2.1" href="#ftn.R.1.2.1">1</a>
                  <span class="csq">]</span>
               </sup>.</p>
            <section id="R.1.3" class="sect1">
               <div class="section-titlepage">
                  <h2>First Section</h2>
               </div>
               <div class="content">
                  <p>A paragraph.</p>
                  <p>This paragraph contains a footnote<sup class="footnote">
                        <span class="osq">[</span>
                        <a id="R.1.3.3.1" href="#ftn.R.1.3.3.1">2</a>
                        <span class="csq">]</span>
                     </sup>.</p>
               </div>
            </section>
            <section id="R.1.4" class="sect1">
               <div class="section-titlepage">
                  <h2>Section Title<sup class="footnote">
                        <span class="osq">[</span>
                        <a id="R.1.4.1.1.1" href="#ftn.R.1.4.1.1.1">3</a>
                        <span class="csq">]</span>
                     </sup>
                  </h2>
               </div>
               <div class="content">
                  <p>A paragraph.</p>
                  <section id="R.1.4.3" class="sect2">
                     <div class="section-titlepage">
                        <h3>Subsection Title</h3>
                     </div>
                     <div class="content">
                        <p>This paragraph contains a footnote<sup class="footnote">
                              <span class="osq">[</span>
                              <a id="R.1.4.3.2.1" href="#ftn.R.1.4.3.2.1">4</a>
                              <span class="csq">]</span>
                           </sup>.</p>
                     </div>
                  </section>
               </div>
            </section>
         </div>
         <section deltaxml:deltaV2="A" id="R.1.3" class="sect1">
            <div class="section-titlepage">
               <h2>First Section</h2>
            </div>
            <p>A paragraph.</p>
            <p>This paragraph contains a footnote<sup class="footnote">
                  <span class="osq">[</span>
                  <a id="R.1.3.3.1" href="#ftn.R.1.3.3.1">2</a>
                  <span class="csq">]</span>
               </sup>.</p>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><section deltaxml:deltaV2="A" id="R.1.4" class="sect1">
            <div class="section-titlepage">
               <h2>Section Title<sup class="footnote">
                     <span class="osq">[</span>
                     <a id="R.1.4.1.1.1" href="#ftn.R.1.4.1.1.1">3</a>
                     <span class="csq">]</span>
                  </sup>
               </h2>
            </div>
            <p>A paragraph.</p>
            <section id="R.1.4.3" class="sect2">
               <div class="section-titlepage">
                  <h3>Subsection Title</h3>
               </div>
               <p>This paragraph contains a footnote<sup class="footnote">
                     <span class="osq">[</span>
                     <a id="R.1.4.3.2.1" href="#ftn.R.1.4.3.2.1">4</a>
                     <span class="csq">]</span>
                  </sup>.</p>
            </section>
         </section><deltaxml:textGroup deltaxml:deltaV2="A"><deltaxml:text deltaxml:deltaV2="A">
         </deltaxml:text></deltaxml:textGroup><div deltaxml:deltaV2="A=B" class="footnotes">
            <hr width="100" align="left" class="footnotes-divider"/>
            <div class="footnote">
               <div class="footnote-wrapper">
                  <div class="footnote-symbol-wrapper">
                     <p>
                        <sup>
                           <span class="osq">[</span>
                           <a href="#R.1.2.1" id="ftn.R.1.2.1">1</a>
                           <span class="csq">]</span>
                        </sup> </p>
                  </div>
                  <div class="footnote-body-wrapper">
                     <p>This is a footnote.</p>
                  </div>
               </div>
            </div>
            <div class="footnote">
               <div class="footnote-wrapper">
                  <div class="footnote-symbol-wrapper">
                     <p>
                        <sup>
                           <span class="osq">[</span>
                           <a href="#R.1.3.3.1" id="ftn.R.1.3.3.1">2</a>
                           <span class="csq">]</span>
                        </sup> </p>
                  </div>
                  <div class="footnote-body-wrapper">
                     <p>This is another footnote.</p>
                  </div>
               </div>
            </div>
            <div class="footnote">
               <div class="footnote-wrapper">
                  <div class="footnote-symbol-wrapper">
                     <p>
                        <sup>
                           <span class="osq">[</span>
                           <a href="#R.1.4.1.1.1" id="ftn.R.1.4.1.1.1">3</a>
                           <span class="csq">]</span>
                        </sup> </p>
                  </div>
                  <div class="footnote-body-wrapper">
                     <p>Boring, I know, but it just tests the fact that a footnote
can occur in a title.</p>
                  </div>
               </div>
            </div>
            <div class="footnote">
               <div class="footnote-wrapper">
                  <div class="footnote-symbol-wrapper">
                     <p>
                        <sup>
                           <span class="osq">[</span>
                           <a href="#R.1.4.3.2.1" id="ftn.R.1.4.3.2.1">4</a>
                           <span class="csq">]</span>
                        </sup> </p>
                  </div>
                  <div class="footnote-body-wrapper">
                     <p>This is yet another footnote.</p>
                  </div>
               </div>
            </div>
         </div>
      </article>
      <script deltaxml:deltaV2="A=B" src="../../resources/js/prism.js"/>
   </body>
</html>