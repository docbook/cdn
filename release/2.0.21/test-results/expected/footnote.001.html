<html xmlns="http://www.w3.org/1999/xhtml"><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><title>Article wrapper</title><meta charset="utf-8" /><meta name="generator" content="DocBook XSL 2.0 Stylesheets V2.0.17" /><link rel="stylesheet" type="text/css" href="../../resources/css/default.css" /><script type="text/javascript" src="../../resources/js/dbmodnizr.js"></script><link href="../../resources/css/prism.css" rel="stylesheet" type="text/css" /><link href="../../resources/css/db-prism.css" rel="stylesheet" type="text/css" /></head><body><article id="R.1" class="article"><header class="article-titlepage"><h2>Article wrapper</h2></header><div class="lists-of-titles"><div class="toc"><p><b>Table of Contents</b></p><ul class="toc"><li><span><a href="#R.1.3">First Section</a></span></li><li><span><a href="#R.1.4">Section Title</a></span><ul class="toc"><li><span><a href="#R.1.4.3">Subsection Title</a></span></li></ul></li></ul></div></div><p>This paragraph contains a footnote<sup class="footnote"><span class="osq">[</span><a id="R.1.2.1" href="#ftn.R.1.2.1">1</a><span class="csq">]</span></sup>.</p><section id="R.1.3" class="sect1"><div class="section-titlepage"><h2>First Section</h2></div><p>A paragraph.</p><p>This paragraph contains a footnote<sup class="footnote"><span class="osq">[</span><a id="R.1.3.3.1" href="#ftn.R.1.3.3.1">2</a><span class="csq">]</span></sup>.</p></section><section id="R.1.4" class="sect1"><div class="section-titlepage"><h2>Section Title<sup class="footnote"><span class="osq">[</span><a id="R.1.4.1.1.1" href="#ftn.R.1.4.1.1.1">3</a><span class="csq">]</span></sup></h2></div><p>A paragraph.</p><section id="R.1.4.3" class="sect2"><div class="section-titlepage"><h3>Subsection Title</h3></div><p>This paragraph contains a footnote<sup class="footnote"><span class="osq">[</span><a id="R.1.4.3.2.1" href="#ftn.R.1.4.3.2.1">4</a><span class="csq">]</span></sup>.</p></section></section><div class="footnotes"><hr width="100" align="left" class="footnotes-divider" /><div class="footnote"><div class="footnote-wrapper"><div class="footnote-symbol-wrapper"><p><sup><span class="osq">[</span><a href="#R.1.2.1" id="ftn.R.1.2.1">1</a><span class="csq">]</span></sup> </p></div><div class="footnote-body-wrapper"><p>This is a footnote.</p></div></div></div><div class="footnote"><div class="footnote-wrapper"><div class="footnote-symbol-wrapper"><p><sup><span class="osq">[</span><a href="#R.1.3.3.1" id="ftn.R.1.3.3.1">2</a><span class="csq">]</span></sup> </p></div><div class="footnote-body-wrapper"><p>This is another footnote.</p></div></div></div><div class="footnote"><div class="footnote-wrapper"><div class="footnote-symbol-wrapper"><p><sup><span class="osq">[</span><a href="#R.1.4.1.1.1" id="ftn.R.1.4.1.1.1">3</a><span class="csq">]</span></sup> </p></div><div class="footnote-body-wrapper"><p>Boring, I know, but it just tests the fact that a footnote
can occur in a title.</p></div></div></div><div class="footnote"><div class="footnote-wrapper"><div class="footnote-symbol-wrapper"><p><sup><span class="osq">[</span><a href="#R.1.4.3.2.1" id="ftn.R.1.4.3.2.1">4</a><span class="csq">]</span></sup> </p></div><div class="footnote-body-wrapper"><p>This is yet another footnote.</p></div></div></div></div></article><script src="../../resources/js/prism.js"></script></body></html>